olexandr.vakulinskiy/HW3/Task1/prime_numbers.rb

```diff
@@ -0,0 +1,54 @@
+require 'matrix'
+class Matrix
+  def print_matrix
+    count = 0
+    each do |number|
+      if count < column_size - 1
+        print number.to_s + ', '
+        count += 1
+      else
+        print number.to_s + ' '
+        count += 1
+      end
+      if count == column_size
+        print "\n"
+        count = 0
+      end
+    end
+  end
+end
+
+class PrimeNumbers
+  attr_reader :matrix
+
+  def initialize(max_num)
+    @num = max_num
+    @matrix = matrix
+    fill_matrix(primes)
+  end
+
+  private
+
+  def primes
+    array = []
+    (1..@num).each do |number|
+      array.push(number) if (1..number).select { |d| (number % d).zero? } == [1, number]
+    end
+    array
+  end
+
+  def fill_matrix(row)
+    rotation_count = row.size - 1
+    count = 0
+    until count > rotation_count
+      @matrix = if count.zero?
+                  Matrix.rows(@matrix.to_a << row)
+                else
+                  Matrix.rows(@matrix.to_a << row.rotate(count))
+                end
```

Comment:
Please, use [ternary operators](https://github.com/rubocop-hq/ruby-style-guide#ternary-operator) in such cases.
```suggestion
      @matrix = count.zero? ? Matrix.rows(@matrix.to_a << row) : Matrix.rows(@matrix.to_a << row.rotate(count))
```